ui/org.eclipse.pde.runtime/src/org/eclipse/pde/internal/runtime/registry/model/Property.java

```diff
@@ -62,8 +62,6 @@ public void setValue(String value) {
 	 * than other properties.</li>
 	 * <li>regular properties are considered "more" than other properties</li>
 	 * </ul>
-	 * </p>
-	 *
```

Comment:
Please make sure that the that the paragraphs are balanced and each opened element has a matching closing tag.